src/map/zone_entities.cpp

```diff
@@ -819,12 +819,16 @@ void CZoneEntities::PushPacket(CBaseEntity* PEntity, GLOBAL_MESSAGE_TYPE message
             }
             case CHAR_INRANGE:
             {
+                // todo: rewrite packet handlers and use enums instead of rawdog packet ids
+                // 30 yalms if action packet and in combat, 50 otherwise
+                const int checkDistanceSq = packet->id() == 0x0028 && PEntity->PAI->IsEngaged() ? 900 : 2500;
```

Comment:
why the engaged check?  i don't think that's part of anything that was said in windower

Reply:
i made poopy